package/windows/pyinstaller.spec

```diff
@@ -127,6 +131,31 @@ def move_silx_view_exe():
 
 move_silx_view_exe()
 
+# Generate license file from current Python env
+def create_license_file(filename: str):
+    import PyQt5.QtCore
+
+    with open(filename, 'w') as f:
+        f.write(f"""
+This is free software.
+
+This distribution of silx is provided under the
+GNU General Public License v3 (https://www.gnu.org/licenses/gpl-3.0.en.html) since it includes PyQt5.
+
+It includes mainy software packages with different licenses:
+
+- Python ({sys.version}): PSF license, https://www.python.org/
+- Qt ({PyQt5.QtCore.qVersion()}): GNU Lesser General Public License v3, https://www.qt.io/
+""")
+
+        for dist in sorted(pkg_resources.working_set, key=lambda d: d.key):
+            license = importlib.metadata.metadata(dist.key).get('License')
+            homepage = importlib.metadata.metadata(dist.key).get('Home-page')
+            info = ", ".join(info for info in (license, homepage) if info)
+            f.write(f"- {dist.project_name} ({importlib.metadata.version(dist.key)}): {info}\n")
+
+create_license_file('LICENSE')

```

Comment:
@mretegan you might be interested in this: it generates the list of all Python packages used (actually the one installed in the env used to make the fat binary) with the version, license and homepage information and use it as the license file.

Reply:
@t20100 Thanks for letting me know about this. Looks nice!